.github/workflows/e2e_parallel_jobs.yaml

```diff
@@ -31,7 +31,7 @@ jobs:
 
       - name: Install dependences
         run: |
-          GO111MODULE="on" go install sigs.k8s.io/kind@v0.15.0
+          GO111MODULE="on" go install sigs.k8s.io/kind@v0.21.0
           curl -LO https://storage.googleapis.com/kubernetes-release/release/v1.23.0/bin/linux/amd64/kubectl && sudo install kubectl /usr/local/bin/kubectl
```

Comment:
can we also upupgrade kubectl to v1.29.0?
refer to: https://github.com/volcano-sh/volcano/pull/3460#issuecomment-2105589681

Reply:
Yes，I will upgrade it.